web/changelog.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "version": "3.8.x",
```

Comment:
This seems unsuitable for release. Maybe we should start working with release branches?